cmd/storage-datatypes.go

```diff
@@ -37,9 +37,18 @@ type DiskInfo struct {
 	Endpoint   string
 	MountPath  string
 	ID         string
+	Metrics    DiskMetrics
 	Error      string // carries the error over the network
 }
 
+// DiskMetrics has the information about XL Storage APIs
+// the number of calls of each API and the moving average of
+// the duration of each API.
+type DiskMetrics struct {
+	APILatencies map[string]string `json:"apiLatencies,omitempty"`
```

Comment:
Nit: Should `APIAvgLatenciesPerCall` It is not a list of latencies but the average of each API invocation.

Reply:
Since its a map @kerneltime its implicit - we don't need to say that itis per call

Reply:
I kept the key name smaller on purpose